providers/redfish/firmware.go

```diff
@@ -409,6 +417,24 @@ func (c *Conn) FirmwareInstallStatus(ctx context.Context, installVersion, compon
 	switch {
 	case strings.Contains(vendor, constants.Dell):
 		task, err = c.dellJobAsRedfishTask(taskID)
+	case strings.Contains(vendor, constants.Packet) || strings.Contains(vendor, constants.Equinix):
+
+		resp, _ := c.redfishwrapper.Get("/redfish/v1/TaskService/Tasks/" + taskID)
+		if resp.StatusCode != 200 {
+			err = errors.Wrap(
+				bmclibErrs.ErrFirmwareInstall,
```

Comment:
This should be a `ErrFirmwareInstallStatus` error instead
 https://github.com/bmc-toolbox/bmclib/blob/redfish-firmware-installs/errors/errors.go#L67